NetworkIOMeter.c

```diff
@@ -45,7 +45,11 @@ static void NetworkIOMeter_updateValues(Meter* this) {
 
    /* update only every 500ms to have a sane span for rate calculation */
    if (passedTimeInMs > 500) {
+#ifdef IGNORE_VIRTUAL_INTF
+      hasNewData = Platform_getNetworkIO(&data, host->settings->ignoreVirtualNetworkInterfaces);
```

Comment:
I don't like two kinds of function prototypes that depends on a compile time option. Even though the code can compile, the braces are unbalanced and would very likely confuse IDEs.

Another point is, you might not need to pass the boolean setting as a function argument. I suggest making it a member of the `NetworkIOData` (`bool ignoreVirtualInterfaces;`), and let `NetworkIOMeter_updateValues` set it before calling `Platform_getNetworkIO`.